src/qml/imports/Theme/QfToolButtonDrawer.qml

```diff
@@ -0,0 +1,79 @@
+import QtQuick 2.14
+import QtQuick.Controls 2.14
+
+import Theme 1.0
+
+Container {
+  id: container
+
+  property string name: ''
+  property real size: 48
+  property bool downward: true
+  property bool vertical: true
```

Comment:
```suggestion

  enum Direction {
    Up,
    Down,
    Left,
    Right
  }

  property string name: ''
  property real size: 48
  property int direction: Direction.Down
```

I think this is a bit cleaner when we use it, also will make the code below to a switch/case manner which IMO will make it much easier to understand instead of contant boolean checks.